processor/eval/ledger_for_evaluator.go

```diff
@@ -151,5 +151,5 @@ func (l LedgerForEvaluator) LatestTotals() (ledgercore.AccountTotals, error) {
 
 // BlockHdrCached is part of go-algorand's indexerLedgerForEval interface.
 func (l LedgerForEvaluator) BlockHdrCached(round basics.Round) (bookkeeping.BlockHeader, error) {
-	panic("not implemented")
+	return l.Ledger.BlockHdrCached(round)
```

Comment:
Yeah I also found this was necessary in my state proofs PR, but I wasn't sure if it was due to upstream changes not yet in go-algorand/master.